build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Replace with our standard Copyright (from mongo-java-driver, except change start year to 2024)